test/LoggingSpec.hs

```diff
@@ -39,27 +22,27 @@ spec = describe "LoggingSpec" $ do
     it "can generate a main log message" $ do
       stackParams <- runExceptT . getStackParameters $ "test/valid.yaml"
       let descriptions = fromRight' stackParams
-      let accountID = "478156153062"
+      let accountID = "478156153XXX"
```

Comment:
Was this a real account ID? 😱 🤦

Reply:
No idea, it's been in there for a long time so hopefully it's all good. I only came across it because a recommit hook fired and said is was no es bueno. 😬